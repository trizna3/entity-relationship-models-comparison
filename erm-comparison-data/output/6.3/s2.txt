matches-match
rounds-EMPTY
players-players
teams-teams
points-statistics
