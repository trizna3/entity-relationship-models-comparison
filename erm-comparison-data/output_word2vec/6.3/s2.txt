rounds-round
players-players
points-statistics
matches-match
teams-teams
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-round-match
