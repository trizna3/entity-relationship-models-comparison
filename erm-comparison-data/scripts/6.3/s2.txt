entity set teams {
	"name"
};

entity set players {
};

entity set match {
	"round"
};

entity set statistics {
	"goal"
	"assist"
	"time"
	"period"
};

association  between 
{
	teams ONE 
	match MANY
}{};

association between 
{
	teams ONE 
	match MANY
}{};

association between 
{
	teams ONE 
	players MANY
}{};

association "assistant" between 
{
	players ONE 
	statistics ONE
}{};

association "scored" between 
{
	players ONE 
	statistics ONE
}{};

association between 
{
	match ONE 
	statistics ONE
}{};