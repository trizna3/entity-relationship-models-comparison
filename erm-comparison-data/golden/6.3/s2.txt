teams-teams
players-players
matches-match
points-statistics
rounds-empty
FLIP_CARDINALITY:E-matches-rounds
CONTRACT_11_ASSOCIATION:E-matches-rounds